src/AppInstallerCLICore/Commands/PinCommand.cpp

```diff
@@ -168,9 +164,7 @@ namespace AppInstaller::CLI
 
     void PinRemoveCommand::Complete(Execution::Context& context, Args::Type valueType) const
     {
-        context <<
-            Workflow::OpenSource() <<
-            Workflow::OpenCompositeSource(Repository::PredefinedSource::Installed);
+        context << Workflow::OpenSourceFor(OperationType::Pin);

```

Comment:
I forgot to change this. Maybe we should also conditionally open only the available source here